log-collector-script/linux/eks-log-collector.sh

```diff
@@ -491,6 +491,19 @@ get_networking_info() {
   # configure-multicard-interfaces
   timeout 75 journalctl -u configure-multicard-interfaces > "${COLLECT_DIR}"/networking/configure-multicard-interfaces.txt || echo -e "\tTimed out, ignoring \"configure-multicard-interfaces unit output \" "
 
+  # test some network connectivity
+  timeout 75 ping -A -c 10 amazon.com > "${COLLECT_DIR}"/networking/ping_amazon.com.tzt
+  timeout 75 ping -A -c 10 public.ecr.aws > "${COLLECT_DIR}"/networking/ping_public.ecr.aws.txt
+
+  if [[ -e "${COLLECT_DIR}"/kubelet/kubeconfig.yaml ]]; then
+    API_SERVER=$(grep server: "${COLLECT_DIR}"/kubelet/kubeconfig.yaml | sed 's/.*server: //')
+    for i in $(seq 5); do
+      echo -e "curling ${API_SERVER} ($i of 5) $(date --utc +%FT%T.%3N%Z)\n\n"  >> ${COLLECT_DIR}"/networking/curl_api_server.txt"
+      timeout 75 curl -v --cacert /etc/kubernetes/pki/ca.crt "${API_SERVER}" >> ${COLLECT_DIR}"/networking/curl_api_server.txt" 2>&1
```

Comment:
nit: pull the `certificate-authority` out of the kubeconfig like you did `API_SERVER`?